### DISCUSSION TITLE
Need Help installing arm docker

### DISCUSSION THREAD

**Discussion by Emrite2** [automatic-ripping-machine/automatic-ripping-machine#1038] [Q&A]
Can anyone tell me what to do when this happens when i try to run the command ./start_arm_container.sh? 

![Screenshot 2024-01-19 172113](https://github.com/automatic-ripping-machine/automatic-ripping-machine/assets/157180587/85a628e5-e41a-47ab-9c75-40bc47c54101)

This is my first time using Linux , so I am very confused on what to do.

**Comment by 1337-server** [comment]
The file/script you are trying to run need edited manually, you will need to select where the files need to go and then add the location to the script. All the replacements should be surrounded by the <> marks.
For example.
```
    -v "<path_to_arm_user_home_folder>:/home/arm" \
    -v "<path_to_music_folder>:/home/arm/music" \
    -v "<path_to_logs_folder>:/home/arm/logs" \
    -v "<path_to_media_folder>:/home/arm/media" \
    -v "<path_to_config_folder>:/etc/arm/config" \
```
You will also need to include the correct ID for the arm user.
